docs/docs/features/request-validation.md

```
@@ -145,6 +145,10 @@ You will need to take care to ensure read-only fields are not modified. It's up
 
 Write-only fields, if stored in a datastore, can be combined with `omitempty` and then set to the zero value in handlers, or filtered out via datastore queries or projection. They can also be kept out of the datastore altogether but used to compute values in fields that will get stored.
 
+!!! info "Note"
+
+    If a write-only field needs to be required on the request but the same struct is re-used in the response, you can use `json:"name,omitempy"` with `required:"true"`.
```

Markdownlint

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)